Assets/Scripts/WorkPuzzle/WordDisplay.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+public class WordDisplay : MonoBehaviour
+{
+    public TextMeshPro text;
+    public GameObject loseText;
+    //speed in which the text will fall down the screen
+    private float speed = 10f;
+
+    public SpawnText spawnText;
+
+    public void SetText(string word)
+    {
+        text.text = word;
+    }
+
+    public void DeleteChar()
+    {
+        text.text = text.text.Remove(0, 1);
+        text.color = Color.green;
+    }
+
+    public void DeleteWord ()
+    {
+        Destroy(gameObject);
+    }
+
+    private void FixedUpdate()
```

Comment:
I think we should have one update loop that keeps track of a list of the words and moves all of them each update, rather than all of the words having their own update loops, which can bog down performance.